source/manuals/configure-service-broker.html.md.erb

```diff
@@ -1,41 +1,45 @@
 ---
-title: Monitoring your apps with Prometheus
+title: How to monitor your apps with Prometheus
 ---
 
 # <%= current_page.data.title %>
 
-Prometheus uses service discovery to determine which apps to monitor, for PaaS apps this requires users to follow these steps:
+[Prometheus][] uses service discovery to decide what it monitors, so for apps running on [GOV.UK PaaS][] you'll need to:
 
-1. Granting Prometheus read-only access to your space
-2. Indicating which apps Prometheus should monitor by binding apps with the service broker
+* grant Prometheus read-only access to your PaaS spaces
+* bind your apps with the Prometheus service broker
 
-### Granting Prometheus read-only access to your PaaS spaces
+### Grant Prometheus read-only access to your PaaS spaces
 
-Giving prometheus-for-paas the SpaceAuditor role allows it to monitor each instance of your app and respond to events (start/stop/scaling).
+By giving prometheus-for-paas the [SpaceAuditor][] role you allow it to monitor each instance of your app and respond to events (start/stop/scaling).
 
 `cf set-space-role prometheus-for-paas@digital.cabinet-office.gov.uk <org-name> <space-name> SpaceAuditor`
 
-### Using the Service Broker to bind Prometheus
+### Use the service broker to bind Prometheus
 
-Our Prometheus service broker can be found in the PaaS marketplace.
+The Prometheus service broker can be found in the PaaS marketplace.
 
 ```
 ❯ cf marketplace
 service          plans        description
 gds-prometheus   prometheus   GDS internal Prometheus monitoring alpha https://reliability-engineering.cloudapps.digital/#metrics
 ```
 
-If you are unable to see `gds-prometheus` in the output of `cf marketplace` please get in touch in the [#reliability-eng Slack channel].
+If you're unable to see `gds-prometheus` in the output of `cf marketplace` please contact us through the [#reliability-eng Slack channel].
 
-These steps will create a Prometheus service within your space and allow you to bind it to running apps in your space.
+Once you have the service broker you can create a Prometheus service within your PaaS space and allow it to bind to apps running there. Do this by following these steps:
```

Comment:
Do you think it'd be helpful to include a link to the relevant Cloud Foundry doc here?